library/Zend/Db/Select.php

```diff
@@ -700,7 +700,7 @@ public function getPart($part)
      *
      * @param integer $fetchMode OPTIONAL
      * @param  mixed  $bind An array of data to bind to the placeholders.
-     * @return PDO_Statement|Zend_Db_Statement
+     * @return PDOStatement|Zend_Db_Statement
```

Comment:
If this is https://www.php.net/manual/en/class.pdostatement.php?

then you must either import it:
- `use PDOStatement`
- or fqdn `\PDOStatement`

and in any case, is that even correct, the code always returns Zend_Db_Statement there.

the class may be compatible with (some) of PDOStatement methods,
but it's not accurate:
- https://framework.zend.com/manual/1.12/en/zend.db.statement.html